### PR TITLE
Christopher foster day2 stretch

### PR DESCRIPTION
**Pull request by catslanderer**

### CODE REVIEW

File: client/src/components/User.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const User = ({ user }) => {
+  return (
+    <div className='user'>
+      <h2>{user.id}</h2>
+      <h2>{user.created_at}</h2>
+      <h2>{user.username}</h2>
+      <h2>{user.password}</h2>
```
Comment (DaftBeowulf) [inline client/src/components/User.js:9]: even though it's a hash, are you sure you want to be displaying this in the app?

File: client/src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+import Home from './Home';
+import SignUp from './SignUp';
+import SignIn from './SignIn';
+import Users from './Users';
+
+const App = ({ history }) => {
+  const signOut = () => {
+    localStorage.removeItem('token');
+    history.push('/signin');
```
Comment (DaftBeowulf) [inline client/src/components/App.js:12]: redirecting users after they log out, nice

**Review by DaftBeowulf — COMMENTED** [review]
Good job on this extra stretch Chris--consider my suggestions left on your previous branch to find some way to display the received error messages a user could run into when attempting to sign in or register, or at least throw up an alert to that effect, as attempting to do either with faulty info doesn't tell the user anything. Overall this was a great project to tie everything you learned last week together :+1:

File: client/src/components/User.js (hunk 1 above)
Comment (catslanderer) [reply client/src/components/User.js:9]: Yeah - I wouldn't display a password in a real app, haha.

**Pull request merged**